spec/spec_helper_models.rb

```diff
@@ -1,5 +1,5 @@
 RSpec.configure do |config|
-  config.before(:each) do
+  config.before do
```

Comment:
Is this a stylistic choice because :each is the default? Personally I think I slightly prefer the explicitness of `:each`.

Reply:
Two yes:

- `:example` is the new name of `:each`
- `:example` is the default